.github/workflows/mfem-analysis.yml

```diff
@@ -19,7 +19,7 @@ on:
     branches:
       - master
       - next
-  pull_request:
+  pull_request: []
```

Comment:
What is the difference between `pull_request: []` and `pull_request:`?

Reply:
It's just more explicit to say it's empty. I have had issues with other event triggers not providing it (e.g., dispatch event) so I tend to apply it to be consistent. But this is probably not changing anything so feel free to not use it.